modulefiles/wcoss2/fv3_debug

```diff
@@ -0,0 +1,42 @@
+#%Module
+
+proc ModulesHelp {} {
+  puts stderr "\tcit - loads modules required for building and running UFS Model on the Dell side of WCOSS"
+}
+
+module-whatis "loads UFS Model prerequisites on Venus and Mars"
+
+module unload cpe-cray cce
+module load cpe-intel intel
+
+module load cmake/3.17.3
+setenv CMAKE_C_COMPILER cc
+setenv CMAKE_CXX_COMPILER CC
+setenv CMAKE_Fortran_COMPILER ftn
+setenv CMAKE_Platform wcoss2
+
```

Comment:
One could just add `I_MPI_LINK` environment variable in the debug modulefile and eliminate the hard-wired compile time flags.